### ISSUE TITLE
As a Reg agent, Registar or National Reg I want to see a detailed view of the action performed on a record

### ISSUE THREAD

**Issue by jpye-finch** [opencrvs/opencrvs-core#2426]
### Description
- Can a Field Agent see the detailed view?? Will need to see comments and reasons for rejections if in the future we want them to make updates... @anny320 @eduffus 
- adds a model that shows a detailed view of an action
- clicking on an action opens up the model
- content for each model is listed in this [table](https://www.notion.so/opencrvs/Audit-5ea0204d2b6c40858e772028784db864)

### Acceptance criteria

GIVEN I am on the Record Audit page
WHEN I click on an action
THEN open a model with the detailed view of that action

### Design
https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/00.OpenCRVS-Specifications?node-id=1778%3A27117

### Dev-tasks

- [ ]  Launch a RecordAuditDetailModal to display information from the operation for the selected action.

- [ ] Need to address following review comment in this ticket
https://github.com/opencrvs/opencrvs-core/pull/2600#discussion_r817419076

- [ ] Reuse OperationHistory instead of Hostory in record Audit [Comment](https://github.com/opencrvs/opencrvs-core/pull/2598#discussion_r815604783)

**Comment by euanmillar** [comment]
Blocked by #2459

**Comment by SyedaAfrida** [comment]
**Bug:**

Updated declaration dialogue not showing correct
![2](https://user-images.githubusercontent.com/94350336/161708967-66924b45-5759-403b-a7f1-0b8242d4f650.png)

Informant, father & mother's NID and Name is mixed up in Update declaration dialogue.

https://user-images.githubusercontent.com/94350336/161709055-4c358b4f-9ae5-460f-8482-e04c59a1ab10.mp4

**Comment by yeasinhossain-dsi** [comment]
@jpye-finch in the video you will see there `National ID`, `First Name`, `Last Name` etc are shown multiple times in the history modal. As the Mother, Father & informant all have these fields, we need to show in the modal which on represents who. 

Will it be fine if we show something like: `Father: National ID`. The grammar here is `Section Name: Field Name`.

**Comment by saima-alam** [comment]
@anny320 The mentioned bug in Sayda's comment is not fixed yet. Created a new ticket for the bug #3252
